### PR TITLE
Update zh_CN translation

### PR DESCRIPTION
**Pull request by l2dy**
* **What does this PR do?**
@yuzibo @flatcap Last part of zh_CN translation!
@yuzibo ~~It's still WIP, but I'd appreciate a review.~~ Ready for review!
* **Are there points in the code the reviewer needs to double check?**

* **Screenshots (if relevant)**

* **Does this PR meet the acceptance criteria?** (This is just a reminder for you,
  this section can be removed if you fulfill it.)

   - Documentation created/updated (you have to edit
     [doc/manual.xml.head](https://www.github.com/neomutt/neomutt/blob/master/doc/manual.xml.head)
     for that)

   - All builds are passing

   - Added [doxygen code documentation](https://neomutt.org/dev/doxygen)
     [syntax](http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html)

   - Code follows the [style guide](https://neomutt.org/dev/coding-style)

* **What are the relevant issue numbers?**

### CODE REVIEW

**Comment by flatcap** [comment]
Awesome!
Thanks for all your hard work!

File: po/zh_CN.po [hunk 1]
```diff
@@ -2739,6 +2738,41 @@ msgid ""
 "  -z            Open the first or specified (-f) mailbox if it holds any message\n"
 "                or exit immediately with exit code 1 otherwise"
 msgstr ""
+"选项：\n"
+"  --            使 NeoMutt 停止选项解析并将剩余参数全部当作地址的特殊参数\n"
```
Comment (yuzibo) [inline po/zh_CN.po]: 特殊参数使 NeoMutt 停止选项解析并将剩余参数作为地址即使以dash开头。\n  ?（我不确定dash）

File: po/zh_CN.po [hunk 2]
```diff
@@ -6003,7 +6037,7 @@ msgstr "已中止未修改过的信件。"
 
 #: send.c:1784
 msgid "No crypto backend configured.  Disabling message security setting."
-msgstr "没有配置加密后端。邮件安全设置已禁用。"
+msgstr "没有配置加密后端。信件安全设置已禁用。"
```
Comment (yuzibo) [inline po/zh_CN.po]: 信件安全设置正在禁用。
我觉得这样才是准确的。

Comment (l2dy) [reply po/zh_CN.po]: The operation "disable message security setting" is instant, so "已禁用信件安全设置" (I reordered this) should be fine.

**Pull request merged**